src/EnergyPlus/UnitarySystem.cc

```diff
@@ -125,49 +125,17 @@ namespace UnitarySystems {
                               bool const ZoneEquipment,
                               Real64 &sysOutputProvided,
                               Real64 &latOutputProvided)
-    {
-        simulateSys(state,
-                    Name,
-                    FirstHVACIteration,
-                    AirLoopNum,
-                    CompIndex,
-                    HeatActive,
-                    CoolActive,
-                    ZoneOAUnitNum,
-                    OAUCoilOutTemp,
-                    ZoneEquipment,
-                    sysOutputProvided,
-                    latOutputProvided);
-    }
-
-    void UnitarySys::simulateSys(EnergyPlusData &state,
-                                 std::string_view Name,
-                                 bool const FirstHVACIteration,
-                                 int const AirLoopNum,
-                                 int &CompIndex,
-                                 bool &HeatActive,
-                                 bool &CoolActive,
-                                 int const ZoneOAUnitNum,
-                                 Real64 const OAUCoilOutTemp,
-                                 bool const ZoneEquipment,
-                                 Real64 &sysOutputProvided,
-                                 Real64 &latOutputProvided)
```

Comment:
Clean up started here and then included a lot of recent suggestions for improvement. This function was added when sysOutputProvided and latOutputProvided were optional arguments and is no longer needed.